addons/default/visiosoft/advs-module/resources/views/list/partials/list-filter.twig

```diff
@@ -1,4 +1,16 @@
-<form action="{{ url_route('visiosoft.module.advs::list') }}" method="get">
+{% if 'cat' in param|keys %}
+    {% set catId = entries('cats', 'category').find(param['cat']) %}
+    {% set cityId = null %}
+    {% if 'city' in param|keys %}
+        {% set citiesArray = param['city'][0]|split(',') %}
+        {% if count(citiesArray) is same as(1) %}
+            {% set cityId = entries('location', 'cities').find(param['city'][0]) %}
+        {% endif %}
+    {% endif %}
+    <form action="{{ url_route('adv_list_seo', [catId.slug, cityId.slug]) }}" method="get">
```

Comment:
is it working with customfields parameters? @Diatrex

Reply:
yes